stock_request/models/res_company.py

```diff
@@ -12,3 +12,11 @@ class ResCompany(models.Model):
     stock_request_allow_virtual_loc = fields.Boolean(
         string="Allow Virtual locations on Stock Requests"
     )
+    stock_request_confirm_user_permission = fields.Selection(
+        selection=[
+            ("user", "User"),
+            ("manager", "Manager"),
+        ],
+        default="user",
```

Comment:
```suggestion
        default="manager",
```

Reply:
Nope as this changes the base behaviour from the beginning of this module

Reply:
Yes, but that's the intended behavior. It's a nonsense to have 2 steps (submit and confirm), when both profiles can make by default both. We'll bump major version.

Reply:
> We'll bump major version.

Yes, but it will be applied on further version. I think about migrations

Reply:
We can put a migration script for those coming from previous versions that preserve the other, but I really think it's a mistake to let it as it was before. If Jordi/Kitti thinks the same, we can even remove this mechanism and let the new default only.

Reply:
OK, I see the problem. The new module `stock_request_submit` is the one to fix:

- It should change the button "Confirm" to launch the method to put the request into submit.
- The button submit should call the method `confirm`.
- This button is the one to be protected by manager permission.